pkg/types/installconfig.go

```diff
@@ -112,6 +113,16 @@ func (c *InstallConfig) ClusterDomain() string {
 	return fmt.Sprintf("%s.%s", c.ObjectMeta.Name, c.BaseDomain)
 }
 
+// ClusterPrivate returns true if this is a private cluster
+func (c *InstallConfig) ClusterPrivate() bool {
```

Comment:
I don't think we should be adding these functions to the public API of the install-config.

Reply:
removed.